inst/htmljs/animint.js

```diff
@@ -2096,7 +2096,7 @@ var animint = function (to_select, json_file) {
 	        .attr("y", 14)
           .style("fill", function(d){return d["textcolour"]||1;})
 	        .style("text-anchor", "middle")
-          .attr("font-size", function(d){return d["textsize"]||1;})
+          	.attr("font-size", function(d){return d["textsize"]||1;})
 	        .text("a");
```

Comment:
indentation still does not match ...

Reply:
Sorry, it must be something wrong with my vscode. I just fixed some of the other code I wrote, is that ok?